### DISCUSSION TITLE
About the API version

### DISCUSSION THREAD

**Discussion by Kiko123cd** [SFDO-Community/declarative-lookup-rollup-summaries#1349] [Q&A]
Could you tell me the API version of the　version number　2.5？
Thank You

**Comment by aheber** [comment]
@Kiko123cd, I think most of the Apex classes are at v37.0 but the triggers are deployed at v32.0 if I'm reading the files correctly.

https://github.com/SFDO-Community/declarative-lookup-rollup-summaries/tree/d42fbb8786c54e229b548fcfab1df2498468b7bb
